apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/StorageFixture.cs

```diff
@@ -288,6 +288,41 @@ private void CreateAndPopulateReadBucket()
 
         internal void UnregisterBucket(string bucket) => _bucketsToDelete.Remove(bucket);
 
+        internal async Task EventuallyAsync(Func<Task> action, TimeSpan delayUntilGuaranteedConsistency)
+        {
+            // We don't want to continue retrying past this point in time
+            // as that's meant to be the time for the change to be eventually consistent.
+            DateTimeOffset shouldSucceedAfter = DateTimeOffset.UtcNow
+                + delayUntilGuaranteedConsistency
+                // We add 3 seconds just to be on the safe side of clock skews.
```

Comment:
Clock skews shouldn't affect this, because it's all relative to the same (local) clock, right? I think we *should* be able to get rid of that. It doesn't do any harm though...